### PR TITLE
P0/trash notes

### PR DESCRIPTION
**Pull request by purvasharmadev**
## Added Trash feature

- user can move notes to trash

### CODE REVIEW

File: src/Component/ModalForm/trash-modal.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import "./note-modal.css";
+import { useNotes } from "../../Context/notes-context";
+import { useTrash } from "../../Context/trash-context";
+
+export function TrashModal({ closeModal, id }) {
+  const { deleteNotes } = useNotes();
+  const { postNotesToTrash} = useTrash();
+
+  const postToTrashHandler = (id) => {
+    console.log("from modal ", id);
```
Comment (dev-prabhat) [inline src/Component/ModalForm/trash-modal.js:10]: remove console.log

File: src/Pages/Notes/notes.js [hunk 2]
```diff
@@ -56,9 +70,10 @@ function Notes() {
                     </p>
 
                     <p
-                      onClick={() => {
-                        deleteNotes(item._id);
-                      }}
+                      // onClick={() => {
+                      //   deleteNotes(item._id);
+                      // }}
```
Comment (dev-prabhat) [inline src/Pages/Notes/notes.js:75]: remove commented code

**Review by dev-prabhat — APPROVED** [review]
Looks go to me , you can merge this branch

**Review by srishtigautam21 — APPROVED** [review]
good to merge

**Pull request merged**